appcues/src/main/java/com/appcues/ui/AppcuesViewModel.kt

```diff
@@ -109,11 +111,7 @@ internal class AppcuesViewModel(
     }
 
     fun onActions(actions: List<ExperienceAction>, interactionType: InteractionType, viewDescription: String?) {
-        uiState.value.let { state ->
-            if (state is Rendering) {
-                experienceRenderer.process(state.experience.renderContext, actions, interactionType, viewDescription)
-            }
-        }
+        actionProcessor.process(renderContext, actions, interactionType, viewDescription)
```

Comment:
this is how it was before, instead of calling experienceRenderer we can leave actionProcessor here for now since the process call had the same signature.

Reply:
Is it a concern that it is no longer checking if the uiState is Rendering? What do you mean by - how it was before?

Reply:
in one of the last PRs this was just a single line calling the processor, i changed that but it was not really necessary as the method signature remained the same for no extra benefit, actionProcessor also requires renderContext as its not tied to a single StateMachine. not sure if we are going to change that but for now we could just continue using the actionProcessor here instead of experienceRenderer.